bom/runtime/pom.xml

```diff
@@ -2571,6 +2572,26 @@
             </dependency>
 
             <!-- Spring -->
+            <dependency>
+                <groupId>me.snowdrop</groupId>
```

Comment:
Shouldn't they be under the `io.quarkus` groupId?

Reply:
As finally it was decided to hosted under snowdrop github organization I took the same `groupId` than we have in the snowdrop-bom, it seems confusing to me if we choose `io.quarkus` for something under `snowdrop`, no?

For the version I like using the Spring one, yes. WDYT @geoand ?

Reply:
For me personally it doesn't make a difference. I can go either way. Just keep in mind that if we end up with `io.quarkus`, @gsmet you'll most likely be the one doing the releases (or grant me access perhaps).
If we end up with `snowdrop` then we'll be doing the releases and probably grant you access as well.

Reply:
If we got with `*.snowdrop` group id, it might make sense to have `dev.snowdrop` instead.

Reply:
I think it would be better to have these modules on the quarkus side. The release process should not be a big burden and we can more easily test future releases and so on. We can grant @geoand (or someone else) the permission to push artifacts on `io.quarkus`.

Reply:
That's fine with me

Reply:
100% agree